tests/unit/QuantType.spec.js

```diff
@@ -397,4 +397,94 @@ describe('QuantType.vue', () => {
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('stockPlateReader - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'stockPlateReaderV2'}})
+      expect(quantType.name).toEqual('Stock - Plate Reader - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('workingDilutionPlateReader - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'workingDilutionPlateReaderV2'}})
+      expect(quantType.name).toEqual('Working Dilution - Plate Reader - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('normalisation - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'normalisationV2'}})
+      expect(quantType.name).toEqual('Normalisation - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('RNAStock - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'RNAStockV2'}})
+      expect(quantType.name).toEqual('RNA - Stock - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('RNALibrary - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'RNALibraryV2'}})
+      expect(quantType.name).toEqual('RNA - Library - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('chromiumSingleCellcDNA - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'chromiumSingleCellcDNAV2'}})
+      expect(quantType.name).toEqual('Chromium Single Cell - cDNA - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('chromiumSingleCellLibrary - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'chromiumSingleCellLibraryV2'}})
+      expect(quantType.name).toEqual('Chromium Single Cell - Library - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('duplexSeqALLib - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'duplexSeqALLibV2'}})
+      expect(quantType.name).toEqual('Duplex Seq AL Lib - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('duplexSeqLibrary - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'duplexSeqLibraryV2'}})
+      expect(quantType.name).toEqual('Duplex Seq Library - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('heron96cdna - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'heron96cdnaV2'}})
+      expect(quantType.name).toEqual('Heron 96 cDNA - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+
+    it('heron384cdna - V2', () => {
+      const quantType = new cmp({propsData: { quantType: 'heron384cdnaV2'}})
+      expect(quantType.name).toEqual('Heron 384 cDNA - v2')
+      expect(quantType.qcResults.assay.version).toEqual('v2.0')
+    })
+  })
 })
```

Comment:
Similar for this file not ending in a new line character.  There's an option in VSCode to have it automatically end your files with a new line character so you don't have to think about it.